src/client/current-config.coffee

```diff
@@ -14,6 +26,8 @@ getRemoteUrl = (config)->
   url += "&debug_remote=true"   if config.debugRemote
   url += "&access_token=#{config.accessToken}" if config.accessToken?
   url += "&user_hash=#{config.userHash}" if config.userHash != undefined
+  url += "&_bid=#{identifiers.bid}" if identifiers?.bid
+  url += "&_sid=#{identifiers.sid}" if identifiers?.sid
```

Comment:
I thought we decided to rename _bid to device_id. I do not think that prefixing those params with underscore is needed.